docs/output.md

```diff
@@ -26,8 +29,8 @@ The files in the BlobDir dataset which is used to create the online interactive
 <summary>Output files</summary>
 
 - `blobtoolkit/`
-  - `<accession>/`
-    - `*.json.gz`: files generated from genome and alignment coverage statistics
+  - `<assembly-name>/`
+    - `*.json.gz`: files generated from genome and alignment coverage statistics.
```

Comment:
I only can see these output when running `test_raw` profile, optional or not?

Reply:
These should be **always** there. Can you show me the output directory ?

Reply:
It is OK now. Probably my previous run failed somehow.